lua/jslog/plugins/ibl.lua

```diff
@@ -0,0 +1,12 @@
+local highlight = {
```

Comment:
Would be nice to acknowledge that this is a bit of a hack given that I don't really understand the syntax here.

Reply:
Done